translations/wesnoth-A_New_Order/cs.po

```diff
@@ -331,35 +333,39 @@ msgid ""
 "but only in vegetated terrains."
 msgstr ""
 
+# Nevím, jak je to s tím velkým písmenem.
```

Comment:
These next few strings (the ones from `units/macros/amla.cfg`) are all capitalized because they're AMLAs; I've added caret hints to them in [cooljeanius/A_New_Order@6b3f7bd](https://github.com/cooljeanius/A_New_Order/commit/6b3f7bdf09d32d1db1c5d8fce2768528902aaa6c), which I'll include in a PR later